src/onboarding/steps.ts

```diff
@@ -232,18 +226,9 @@ export function _getStepInfo({ firstScreenInStep, navigator, dispatch, props }:
           } else if (choseToRestoreAccount) {
             popToScreen(Screens.Welcome)
             navigate(Screens.ImportWallet)
-          } else if (showRecoveryPhrase) {
+          } else {
             dispatch(initializeAccount())
             navigate(Screens.ProtectWallet)
-          } else if (skipVerification) {
```

Comment:
I guess we don't need these since the next step is always going to be protect wallet, but what was the scenario where verification is already skipped when the user is on the pincode / biometry screen?

Reply:
I'm not totally certain.. I don't know if the `skipVerification` condition was here defensively for completeness or if it was actually reachable; either way since `showRecoveryPhrase` is effectively always `true` now, we can get rid of any conditions that come after it.

Reply:
> but what was the scenario where verification is already skipped when the user is on the pincode / biometry screen?

`skipVerification` is actually a firebase remote config, not if the user has pressed 'skip' on the Verification screens. It was last used when decentralized phone verification went down for some time.

like joe said though, not necessary here anymore because ProtectWallet is always next